management_api.go

```diff
@@ -48,7 +48,7 @@ func (e *Enforcer) GetAllNamedActions(ptype string) []string {
 
 // GetAllRoles gets the list of roles that show up in the current policy.
 func (e *Enforcer) GetAllRoles() []string {
-	return e.GetAllNamedRoles("g")
+	return e.model.GetValuesForFieldInPolicyAllPTypes("g", 1)
```

Comment:
This is not `PTypes`. Maybe rename it.

Reply:
what shall be a suitable name?

Reply:
`GetValuesForFieldInPolicy` would be OK.

Reply:
there is actually a function of that name, which returns values for a specific type.